csharp/src/SeedLang/Engine.cs

```diff
@@ -64,6 +70,11 @@ public class Engine {
       _vm.VisualizerCenter.Unregister(visualizer);
     }
 
+    // Enables variable tracking.
+    public void EnableVariableTracking() {
```

Comment:
How to turn off this switch? Or, consider a bool parameter?

```
public void EnableVariableTracking(bool enable)
```

Or, consider making the property writable?

Reply:
Remove this function, and make the property writable.